### ISSUE TITLE
R CMD check WARNINGs due to license

### ISSUE THREAD

**Issue by rrrlw** [rcarragh/c212#3]
Previous DESCRIPTION versions list `GPL (>=2)` as the package license; currently, DESCRIPTION refers to the file `LICENSE`, which (I think) is usually reserved for non-standard licenses (the file itself contains GPL-3, a standard FOSS license).

This results in 2 WARNINGs in `R CMD check --as-cran` for c212 on my local machine with the following text:
```
* checking CRAN incoming feasibility ... WARNING
Change to non-FOSS package license.
New license:
  LICENSE
Old license:
  GPL (>= 2)
* checking DESCRIPTION meta-information ... WARNING
Non-standard license specification:
  LICENSE
Standardizable: FALSE
```

IMO, this could be fixed by switching the package license in DESCRIPTION to `GPL-3`; however, `GPL-3 + file LICENSE` could also be appropriate.

Can use [this chapter from the *R Packages* book](https://r-pkgs.org/license.html) as a reference.

Part of [this JOSS review](https://github.com/openjournals/joss-reviews/issues/2706).

**Comment by rcarragh** [comment]
I can replicate the error. 
Some background as to why it happens: when the package was originally released to CRAN in 2017, I specified the licence as GPL (>=2) in the DESCRIPTION file. And it remanied as this up to until I submitted my paper to JOSS. As part of the pre-review process I was asked to include a LICENSE file. I couldn't find a suitable file for GPL >=2 (really multiple licences) so I included the GPL-3 file as the sucessor licence. 

I can fix the issue by following your suggestion of:

License: GPL-3 + file LICENSE 

in the DESCRIPTION file and I've added 

Year: 2020
COPYRIGHT HOLDER: Raymond Carragher
Contact: rcarragh@gmail.com

to the LICENSE file after "END OF TERMS AND CONDITIONS"

This doesn't give me any warnings or notes when when I build the package with "check --as-cran".

Note: this will also affect the "Missing parts of GPL-3 #1" open issue.
